app/src/ui/diff/index.tsx

```diff
@@ -229,11 +231,24 @@ export class Diff extends React.Component<IDiffProps, IDiffState> {
       }
 
       if (this.props.file.status.kind === AppFileStatusKind.Renamed) {
-        return (
-          <div className="panel renamed">
-            The file was renamed but not changed
-          </div>
-        )
+        // Check if it was changed too
+        if (this.props.file.status.renameIncludesModifications)
+        {
+          return (
+            <div className="panel renamed">
+               <Octicon symbol={OcticonSymbol.alert} />
+              The file was renamed and includes changes.
+            </div>
+          )
+        }
+        else
+        {
+          return (
+            <div className="panel renamed">
+              The file was renamed but not changed
+            </div>
+          )
+        }
```

Comment:
```suggestion
          return (
            <div className="panel renamed">
              The file was renamed but not changed
            </div>
          )
```

Unnecessary else statement